components/style/properties/longhand/box.mako.rs

```diff
@@ -277,20 +284,23 @@ ${helpers.single_keyword("clear", "none left right both",
 
 // CSS 2.1, Section 11 - Visual effects
 
-// Non-standard, see https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-clip-box#Specifications
 ${helpers.single_keyword("-servo-overflow-clip-box", "padding-box content-box",
-                         products="servo", animatable=False, internal=True)}
+    products="servo", animatable=False, internal=True,
+    spec="Nonstandard, may be standardized (https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-clip-box)")}
 
 ${helpers.single_keyword("overflow-clip-box", "padding-box content-box",
-                         products="gecko", animatable=False, internal=True)}
+    products="gecko", animatable=False, internal=True,
```

Comment:
Is this really an internal property in Gecko? (from the MDN docs it seems web exposed).

If it is exposed, please address this or file an issue for a followup.

If it isn't, please note it in the `spec=`.